addons/website/static/src/js/widgets/fullscreen_indication.js

```diff
@@ -0,0 +1,50 @@
+odoo.define('website.widgets.FullscreenIndication', function (require) {
+'use strict';
+
+var Widget = require('web.Widget');
+
+var FullscreenIndication = Widget.extend({
+    xmlDependencies: ['/website/static/src/xml/website.xml'],
+    template: 'website.fullscreen_indication', 
+    init: function (parent) {
+        this.visible = false;
+        this.displayTime = 2000;
+        this._super(parent);
+    },
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * Displays the fullscreen indication modal
+     *
+     */
+    show: function () {
+        clearTimeout(this.autofade);
+        if (this.visible) {
+            return this.hide();
+        }
+        this.visible = true
+        this.el.classList.add('transition');
+        this.el.classList.add('visible');
+        var self = this;
+        this.autofade = setTimeout(function () {
+            self.hide(true);
+        }, this.displayTime);
```

Comment:
You can avoid having to use the `self = this` trick by passing an arrow function whenever possible (almost always). Also avoid using var in new code